.github/workflows/update-inputs.yml

```diff
@@ -0,0 +1,26 @@
+name: Update Inputs
+
+on:
+  pull_request:
+    paths:
+      - 'package.json'
+      - 'package-lock.json'
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  update-inputs:
+    name: Update Inputs
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
```

Comment:
we might need to define the branch of the pull request explicitly, but I'm not sure